README.md

```diff
@@ -79,6 +79,22 @@ The debug logger can be configured to be used for more than one library. In exam
 
 ``DEBUG=ibm-cloud-sdk-core:debug,other-lib:debug``
 
+## Cookie Jar Support
+By default, cookies are not supported in the sdk core requests.  If your SDK would benefit from this functionality, simply instantiate a cookie jar and pass it in the object containing configuration options to the `BaseService` class, as show below.
+
+```typescript
+import tough = require('tough-cookie');
+
+class MyClass extends BaseService {
+  constructor(options: MyOptions) {
+    options.jar = new tough.CookieJar();
+    super(options)
+  }
+}
+```
```

Comment:
This documentation feels a little weird to me. SDK developers shouldn't be putting this in their SDK - it should be set by the end user. This documentation is for the SDK developer, so maybe it is helpful to show them how it would work. Perhaps, in this example, creating a new instance of `MyClass` and passing in the cookie as an option.

We eventually want the actual documentation to go into to SDK projects themselves, aka the template.

Reply:
@mkistler and I were talking about this - ultimately cookie jar support is a service-level decision.  Cloudant knows that its users who are on legacy services need this to support their basic authentication, so it makes sense to provide this functionality by default (if they choose) in their SDK, since all users will need to do it.

Reply:
Putting it in the template is not a bad idea, but in the case where the SDK authors automatically configure the cookie jar, it would not be useful.

Reply:
I see. I suppose that makes sense but it that's the case, does there need to be an optional cookie flag in the generator?

Also, there's nothing stopping any user creating a cookie jar at any time with this change. Do we want it to be a "secret" feature (undocumented) unless the service team enables it on a global level automatically?

I just want to make sure I fully understand our approach here